### PR TITLE
edited documentation

### PR DESCRIPTION
**Pull request by jenna-tomkinson**
@gwaybio Made some small edits to the repo, mainly documentation.

### CODE REVIEW

File: 1.segment-data/segment-cell-health-data.ipynb [hunk 1]
```diff
@@ -22465,13 +22467,14 @@
...
     "\n",
+    "# run segmentation on Cell Health data\n",
     "chs.segment_cell_health(load_path, save_path, nuclei_model_specs, cytoplasm_model_specs)"
    ]
   }
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 3.8.13 ('1.segment-data-cell-health')",
+   "display_name": "Python 3.10.6 64-bit",
```
Comment (gwaybio) [inline 1.segment-data/segment-cell-health-data.ipynb]: Can you make sure that you're using the correct conda kernel?

File: 2.extract-features/DP-project-utils.py [hunk 2]
```diff
@@ -237,3 +237,4 @@ def compile_project(
         index_save_path, segmentation_data_path, locations_save_path, object
     )
     print("Done compiling locations!")
+
```
Comment (gwaybio) [inline 2.extract-features/DP-project-utils.py]: 😱

File: 2.extract-features/README.md [hunk 3]
```diff
@@ -10,7 +10,7 @@ We use a [pretrained model](https://github.com/broadinstitute/luad-cell-painting
 [Caicedo et al., 2022](https://www.molbiolcell.org/doi/10.1091/mbc.E21-11-0538) trained this model to extract features from Cell Painting data.
 This model extracts features from all 5 cell-painting channels (DNA, ER, RNA, AGP, Mito).
 
-We use the [metadata file](idr0080-screenA-annotation.csv) from idr-0080-way-pertubation downloaded from [IDR github](https://github.com/IDR/idr0080-way-perturbation/blob/74e537fecaa4690f0c98cb1e9a64b45d103de3e3/screenA/idr0080-screenA-annotation.csv).
+We use the [metadata file](../0.image-download/manifest/idr0080-screenA-annotation.csv) from idr-0080-way-pertubation downloaded from [IDR github](https://github.com/IDR/idr0080-way-perturbation/blob/74e537fecaa4690f0c98cb1e9a64b45d103de3e3/screenA/idr0080-screenA-annotation.csv).
```
Comment (gwaybio) [inline 2.extract-features/README.md:13]: nice catch

File: 2.extract-features/README.md [hunk 4]
```diff
@@ -72,7 +72,7 @@ We use Tensorflow GPU while processing mitocheck data.
 
 ## Step 3: Define Project Paths
 
-Inside the notebook [compile-DeepProfiler-projects.ipynb](compile-DeepProfiler-projects.ipynb), the variables`nuc_project_path` and `cyto_project_path` need to be changed to reflect the desired nuc/cyto DeepProfiler project locations.
+Inside the notebook [compile-DP-projects.ipynb](compile-DP-projects.ipynb), the variables `nuc_project_path` and `cyto_project_path` need to be changed to reflect the desired nuc/cyto DeepProfiler project locations.
```
Comment (gwaybio) [inline 2.extract-features/README.md:75]: 🕵️‍♀️

File: 2.extract-features/compile-DP-projects.ipynb [hunk 5]
```diff
@@ -62358,7 +62358,7 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 3.10.4 64-bit",
+   "display_name": "Python 3.10.6 64-bit",
```
Comment (gwaybio) [inline 2.extract-features/compile-DP-projects.ipynb:62361]: I feel like this kernel is wrong too (it should be a conda env)

**Review by gwaybio — CHANGES_REQUESTED** [review]
Nice catches on these items.

Please correct the conda kernels then this should be good to go.

File: 1.segment-data/segment-cell-health-data.ipynb (hunk 1 above)
Comment (jenna-tomkinson) [reply 1.segment-data/segment-cell-health-data.ipynb]: Woahhh interesting! I didn't know that the kernel shows. Ill add the environment on my end and change the kernel. I just opened it to check, good to know opening it in a different kernel makes an impact.

**Comment by jenna-tomkinson** [comment]
@gwaybio The kernels should be fixed. FYI I don't have permissions to merge PR requests on this machine (as far as I can see).

**Review by gwaybio — CHANGES_REQUESTED** [review]
- Please fix red symbol: https://github.com/WayScience/cell-health-data/pull/9/files#r969804061
- Please nbconvert `.ipynb` files (if not converted already)

Also note that we require one approval prior to merging :)

**Comment by jenna-tomkinson** [comment]
@gwaybio Error should be fixed (turned on "insert final newline" setting) and a new .py file made to reflect the added documentation.

**Comment by gwaybio** [comment]
I still see the red symbol 😱 https://github.com/WayScience/cell-health-data/pull/9/files

There are four empty spaces in the new line at the end, and no new line after the four empty spaces. Maybe you forgot to save the file and add after adjusting `vscode`?

After you fix this, I will approve and merge 💯

**Comment by jenna-tomkinson** [comment]
@gwaybio The issue is fixed. Looks like when I added the line manually, it was automatically indented so Github didn't see it as a new line.

**Review by gwaybio — APPROVED** [review]
Looks great! 💯

**Pull request merged**